lib/board.rb

```diff
@@ -0,0 +1,28 @@
+class Board
+
+    attr_reader :board
+
+    def initialize
+        @board = Array.new(6){Array.new(7, '.')}
+    end
+
+    def display
+        puts 'A B C D E F G'
+        @board.each {|row| puts row.join(' ')}
+    end
+
+    def add_piece(column, piece)
```

Comment:
This looks good. I thought about using rows = board.length -1 and count down to iterate over the rows.

Reply:
Interesting, let's try it out to see if it works more effectively for this method!